python_modules/jupedsim/jupedsim/grid.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+from math import sqrt
+
+
+class Grid:
+    """Class to save points and check for neighbours within a radius
+
+    box : an Axis Aligned Bounding Box where the Grid will be able to save points
+    distance_to_agents : radius in which points are searched for
+    """
+    def __init__(self, box, distance_to_agents):
+        self.box = box
+        self.a_r = distance_to_agents
+        width, height = box[1][0] - box[0][0], box[1][1] - box[0][1]
+        # Cell side length
+        self.c_s_l = distance_to_agents / np.sqrt(2)
```

Comment:
Please add information on the implemented algorithm. For me it is not obvious why sqrt(2) is used here.

Reply:
done

Reply:
Please mention the actual algorithm (e.g. refenrenced by a link)

Reply:
paper of algorithm mentioned